expression/expression.go

```diff
@@ -23,10 +23,21 @@ import (
 	"github.com/pingcap/tidb/context"
 	"github.com/pingcap/tidb/model"
 	"github.com/pingcap/tidb/mysql"
+	"github.com/pingcap/tidb/terror"
 	"github.com/pingcap/tidb/util/codec"
 	"github.com/pingcap/tidb/util/types"
 )
 
+// Error instances.
+var (
+	ErrInvalidOperation = terror.ClassExpression.New(CodeInvalidOperation, "invalid operation")
+)
+
+// Error codes.
+const (
+	CodeInvalidOperation terror.ErrCode = 1
```

Comment:
s/CodeInvalidOperation/codeInvalidOperation
It only uses in the package of expression. It doesn't need to be exported.